### ISSUE TITLE
Model simply returns one of the instance photos. 

### ISSUE THREAD

**Issue by mixbag** [d8ahazard/sd_dreambooth_extension#57]
Tried with 0.000005 and 0.000001. Simply returns one of the input images that I trained for whatever I write in the prompt.

**Comment by rehaldinho64** [comment]
Same here

**Comment by Luke2642** [comment]
This happens around step 1600 for me. Earlier checkpoints are more flexible.

In many dreambooth implementations there are "regularisation images" that prevent overfitting. These are image of your class but not your subject. So images of men, but not your subject Tom Hanks, or images from cartoons but not subject Tintin.

So apparently you need 200, cropped to 512x512, put them in a folder, and put the path in "classification images" and it should prevent overfitting.

I'm just trying it now too, but that's the principle. It's confusing because they textbox label is "classification" not "class regularisation".

**Comment by mixbag** [comment]
> 

Hey thanks, I'll try it. Jst wondering what learning rate did you use and adam epsilon value?

**Comment by mixbag** [comment]
> Just the defaults, LR 0.000005 adam epsilon 1e-8

I tried this with regulaization images but now it is giving me the trained image and the used regularization images output. Eg. If I write lion, it will give me either one of the regularization image(Man) or the trained image.

**Comment by eddjrn** [comment]
This algo happens to me, but it returns variations of the input images not following the prompt at all

**Comment by d8ahazard** [comment]
Appreciate the discussion, but this isn't so much an issue, as a matter of determining the proper LR and steps for the number of input and reg/classification images used. 

As such, I'm going to close it for now. Feel free to check out the readme, I have some links and tips on what settings to use based on the above params. 

Happy hunting!
